test/mclda.jl

```diff
@@ -87,6 +88,12 @@ import Random
     @test withclass_scatter(S) ≈ Sw
     @test betweenclass_scatter(S) ≈ Sb
 
+    covestimator = SimpleCovariance(corrected=true)
+    Sce = multiclass_lda_stats(nc, X, y; covestimator=covestimator)
+
+    @test withclass_scatter(S) ≈ Sw
```

Comment:
Sorry I had missed this. Wouldn't it make sense to test `corrected=false` instead to check that `covestimator` is actually used?

Reply:
You are right that there should be a better test and I've just committed it. It actually checks that the expected output for a non-default estimator is obtained.